src/pages/app/posting_project_user/styles/ProfilePage.css

```diff
@@ -35,9 +35,9 @@
 }
 
 .profile-avatar {
-  height: 150px;
-  width: 150px;
-  border-radius: 75px;
+  height: 175px;
+  width: 175px;
+  border-radius: 87.5px;
```

Comment:
**Global style changes needed**

Reply:
this was 175px on Figma and there was no global variable that equal to this value

Reply:
My bad. Fixed